gateway/utils/graphql/read.go

```diff
@@ -182,6 +182,12 @@ func extractAggregate(v *ast.Field) (map[string][]string, error) {
 			if ok {
 				return nil, utils.LogError(fmt.Sprintf("Cannot repeat the same function (%s) twice. Specify all columns within single function field", functionField.Name.Value), "graphql", "extractAggregate", nil)
 			}
+
+			if functionField.Name.Value == "count" && functionField.SelectionSet == nil {
+				functionMap[functionField.Name.Value] = []string{""}
+				return functionMap, nil
```

Comment:
I think you should use **continue** here, there can other functions also.
Also test it with multiple functions
aggregate {
   count
   sum{
     col_a
     col_b
}
}